seb_openedx/middleware.py

```diff
@@ -137,6 +139,17 @@ def handle_access_denied(self, request, view_func, view_args, view_kwargs, cours
         is_banned, new_ban = ban_user(user_name, course_key, request.user.username)
         is_courseware_view = bool(view_func.__name__ == get_courseware_index_view().__name__)
         context.update({"banned": is_banned, "is_new_ban": new_ban})
+
+        if not is_banned:
+            try:
+                usage_key_string = request.resolver_match.kwargs.get('usage_key_string')
+                url = reverse('jump_to', kwargs=dict(course_id=str(course_key), location=str(usage_key_string)))
+                parsed_url = urllib.parse.urlparse(url)
+                seb_url = parsed_url._replace(scheme="seb", netloc=request.get_host())
+                context.update({"jump_to_link": urllib.parse.urlunparse(seb_url)})
+            except Exception as e:
```

Comment:
Is it possible to catch more specific exception/exceptions?

Reply:
I did not want to risk a 500 error for this. So it either renders a good link or nothing. urllib has a very stable api so the errors may only come from the info we pass to it.